backend/configuration.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
 # windows
-# db_config = {'connection_string': 'sa:Passw12#@127.0.0.1,11433/activy_anonimized?driver=SQL+Server+Native+Client+11.0'}
+db_config = {'connection_string': 'sa:Passw12#@127.0.0.1,11433/activy_anonimized?driver=SQL+Server+Native+Client+11.0'}
```

Comment:
Let's use .env files for environment specific settings. I can introduce this in my branch - overpass api.

Reply:
Cool, much clearer